app/client/liveChat.ts

```diff
@@ -0,0 +1,68 @@
+// <style type='text/css'>
+//     .embeddedServiceHelpButton .helpButton .uiButton {
+//         background-color: #005290;
+//         font-family: "Arial", sans-serif;
+//     }
+//     .embeddedServiceHelpButton .helpButton .uiButton:focus {
+//         outline: 1px solid #005290;
+//     }
+// </style>
+
+// tslint:disable:variable-name
+// tslint:disable:no-object-mutation
```

Comment:
There are multiple instances that need disabling so this was an approach to avoid the noise

Reply:
We do the same in other places so that's fine. Alternatively you can turn the rule back on after you've done what you needed to do. But best test if that works correctly as I remember tslint being weird on those cases. This will ensure we can't miss out the rule for the rest of the file (which is fine now but that might change when someone next touches it)

Reply:
Ah yes I've had a look through the codebase, we mostly use the `// tslint:disable-next-line:no-object-mutation`
and yeah I suppose this file could change a fair bit 👍